kimchi/src/circuits/lookup/constraints.rs

```diff
@@ -532,20 +559,44 @@ pub fn constraints<F: FftField>(configuration: &LookupConfiguration<F>, d1: D<F>
     // and if i % 2 = 1, we enforce that the
     // first element of LookupSorted(i) = first element of LookupSorted(i + 1)
 
-    let s_chunk = (0..(lookup_info.max_per_row + 1))
+    let sorted_size = lookup_info.max_per_row + 1 /* for the XOR lookup table */;
```

Comment:
I would remove `XOR` here, if we're intending to go with a combined lookup table, a la #381.

Reply:
to my previous point, that code would definitely have to change if we start supporting multiple lookup tables, so in the mean time let's make it explicit that it's only working for one table, and for the XOR table :o